src/binder_devmon_if.c

```diff
@@ -150,7 +150,7 @@ binder_devmon_if_io_set_indication_filter(
                 RADIO_IND_FILTER_DATA_CALL_DORMANCY;
         } else if (radio_client_interface(self->client) < RADIO_INTERFACE_1_5) {
             code = RADIO_REQ_SET_INDICATION_FILTER_1_2;
-            value = self->display_on ? RADIO_IND_FILTER_ALL_1_2 :
+            value = self->display_on ? RADIO_IND_FILTER_ALL_1_5 :
```

Comment:
Shouldn't `RADIO_IND_FILTER_ALL_1_5` be below, next to `RADIO_REQ_SET_INDICATION_FILTER_1_5`?

Reply:
Yes, that is a mistake.